[2022-09-25 19:35:59.021] [remoteagent] [info] Extension host agent started.
[2022-09-25 19:35:59.536] [remoteagent] [info] [127.0.0.1][2e932a76][ManagementConnection] New connection established.
[2022-09-25 19:35:59.548] [remoteagent] [info] [127.0.0.1][116b6f33][ExtensionHostConnection] New connection established.
[2022-09-25 19:35:59.561] [remoteagent] [info] [127.0.0.1][116b6f33][ExtensionHostConnection] <1653> Launched Extension Host Process.
[2022-09-25 19:36:02.736] [remoteagent] [info] [127.0.0.1][2e932a76][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[2022-09-25 19:36:03.700] [remoteagent] [info] [127.0.0.1][116b6f33][ExtensionHostConnection] <1653> Extension Host Process exited with code: 0, signal: null.
[2022-09-25 19:36:03.701] [remoteagent] [info] Last EH closed, waiting before shutting down
[2022-09-25 19:36:06.478] [remoteagent] [info] Got delay-shutdown request while in shutdown timeout, delaying
[2022-09-25 19:36:06.478] [remoteagent] [info] Cancelling previous shutdown timeout
[2022-09-25 19:36:06.645] [remoteagent] [info] [127.0.0.1][6e2b93be][ManagementConnection] New connection established.
[2022-09-25 19:36:06.700] [remoteagent] [info] [127.0.0.1][76bc87df][ExtensionHostConnection] New connection established.
[2022-09-25 19:36:06.729] [remoteagent] [info] [127.0.0.1][76bc87df][ExtensionHostConnection] <1751> Launched Extension Host Process.
[2022-09-25 19:36:09.954] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at c._throwIfNoPty (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:23:36316)","    at c.updateTitle (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:23:32064)","    at Object.call (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at u.onPromise (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:15:5851)","    at u.onRawMessage (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:15:5234)","    at /root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:15:4520","    at p.invoke (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at T.fire (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.Q (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:9:20703)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-09-25 19:36:12.925] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at c._throwIfNoPty (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:23:36316)","    at c.updateTitle (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:23:32064)","    at Object.call (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at u.onPromise (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:15:5851)","    at u.onRawMessage (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:15:5234)","    at /root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:15:4520","    at p.invoke (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at T.fire (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.Q (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:9:20703)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-09-25 19:36:12.926] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at c._throwIfNoPty (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:23:36316)","    at c.updateIcon (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:23:32124)","    at Object.call (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at u.onPromise (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:15:5851)","    at u.onRawMessage (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:15:5234)","    at /root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:15:4520","    at p.invoke (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at T.fire (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.Q (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:9:20703)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-09-25 19:41:07.624] [remoteagent] [info] New EH opened, aborting shutdown
[2022-09-25 20:06:52.137] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-09-25 20:07:02.454] [remoteagent] [info] [127.0.0.1][76bc87df][ExtensionHostConnection] The client has reconnected.
[2022-09-25 20:07:03.451] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at c._throwIfNoPty (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:23:36316)","    at c.updateIcon (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:23:32124)","    at Object.call (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at u.onPromise (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:15:5851)","    at u.onRawMessage (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:15:5234)","    at /root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:15:4520","    at p.invoke (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at T.fire (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.Q (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:9:20703)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-09-25 20:07:03.451] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at c._throwIfNoPty (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:23:36316)","    at c.updateTitle (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:23:32064)","    at Object.call (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at u.onPromise (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:15:5851)","    at u.onRawMessage (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:15:5234)","    at /root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:15:4520","    at p.invoke (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at T.fire (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.Q (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:9:20703)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-09-25 20:07:03.463] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at c._throwIfNoPty (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:23:36316)","    at c.updateTitle (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:23:32064)","    at Object.call (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at u.onPromise (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:15:5851)","    at u.onRawMessage (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:15:5234)","    at /root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:15:4520","    at p.invoke (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at T.fire (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.Q (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:9:20703)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-09-25 20:07:07.302] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at c._throwIfNoPty (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:23:36316)","    at c.updateTitle (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:23:32064)","    at Object.call (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at u.onPromise (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:15:5851)","    at u.onRawMessage (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:15:5234)","    at /root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:15:4520","    at p.invoke (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at T.fire (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.Q (/root/.vscode-server/bin/e7f30e38c5a4efafeec8ad52861eb772a9ee4dfb/out/vs/platform/terminal/node/ptyHostMain.js:9:20703)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-09-25 20:07:07.838] [remoteagent] [info] [127.0.0.1][6e2b93be][ManagementConnection] The client has reconnected.
[2022-09-25 22:40:07.705] [remoteagent] [info] [127.0.0.1][6e2b93be][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[2022-09-25 22:40:07.823] [remoteagent] [info] [127.0.0.1][76bc87df][ExtensionHostConnection] <1751> Extension Host Process exited with code: 0, signal: null.
[2022-09-25 22:40:07.823] [remoteagent] [info] Last EH closed, waiting before shutting down
[2022-09-25 22:40:48.343] [remoteagent] [info] Got delay-shutdown request while in shutdown timeout, delaying
[2022-09-25 22:40:48.343] [remoteagent] [info] Cancelling previous shutdown timeout
[2022-09-25 22:40:48.540] [remoteagent] [info] [127.0.0.1][4cd0a93d][ManagementConnection] New connection established.
[2022-09-25 22:40:48.599] [remoteagent] [info] [127.0.0.1][2477111b][ExtensionHostConnection] New connection established.
[2022-09-25 22:40:48.618] [remoteagent] [info] [127.0.0.1][2477111b][ExtensionHostConnection] <4092> Launched Extension Host Process.
[2022-09-25 22:45:48.345] [remoteagent] [info] New EH opened, aborting shutdown
[2022-09-25 22:49:44.385] [remoteagent] [info] [127.0.0.1][4cd0a93d][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[2022-09-25 22:49:44.470] [remoteagent] [info] [127.0.0.1][2477111b][ExtensionHostConnection] <4092> Extension Host Process exited with code: 0, signal: null.
[2022-09-25 22:49:44.470] [remoteagent] [info] Last EH closed, waiting before shutting down
[2022-09-25 22:54:44.472] [remoteagent] [info] Last EH closed, shutting down
